games/types/GameManifest.hpp

```diff
@@ -0,0 +1,28 @@
+/*
+** EPITECH PROJECT, 2024
+** shared-arcade
+** File description:
+** GameManifest
+*/
+
+#pragma once
+#include <string>
```

Comment:
Add a space between `#pragma once` and the `include `
```suggestion
#pragma once

#include <string>
```